docs/sources/features/datasources/loki.md

```diff
@@ -189,14 +189,15 @@ datasources:
     jsonData:
       maxLines: 1000
       derivedFields:
-        # Field with internal link pointing to datasource in Grafana
+        # Field with internal link pointing to datasource in Grafana.
```

Comment:
```suggestion
        # Field with internal link pointing to data source in Grafana.
```

Reply:
Per the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md), "data source" is two words.